archive/frames/serializers.py

```diff
@@ -94,30 +94,6 @@ def create(self, validated_data):
         return frame
 
     def create_or_update_frame(self, data):
-        all_proposals = cache.get('proposal_id_set', [])
```

Comment:
The all proposals cache are used in the [authentication models proposal method](https://github.com/observatorycontrolsystem/science-archive/blob/main/archive/authentication/models.py#L18) as well, so that method will need to be changed to pull them out of the new `all_aggregates` cache now.

Reply:
Good catch. There's some time period where that `all_aggregates` cache could not exist (on first boot) or could be out of date depending on the cronjob schedule. Should I fall back to generating it with a SQL query or just return an empty list? How critical is `get_all_proposals` to the general functioning of the app?

Reply:
since its used to get the profile, its pretty critical for superusers to be able to use the UI at least (I think it doesn't work if the profile times out when logging in). But I guess all superusers are LCO staff, so in that case we can run the task. Maybe if we get an empty list from the cache when a superuser logs in, we could just fall back to immediately run the query?

Reply:
I just went with returning an empty list. Hasn't broken anything. This is a super edge case and will eventually resolve itself so I don't think doing a 30+ sec aggregate is worth it. https://github.com/observatorycontrolsystem/science-archive/pull/50/commits/a3b04d382c68094654a6ddcf832e910e23b244ee